src/projectscene/view/clipsview/clipslistmodel.cpp

```diff
@@ -44,6 +44,9 @@ void ClipsListModel::init()
         Q_UNUSED(time);
         updateItemsMetrics();
     });
+    selectionController()->tracksSelected().onReceive(this, [this](const TrackIdList&) {
+        updateItemsMetrics();
```

Comment:
I think here it is worth updating only those parameters that depend on the selection
am I right?